static/js/publisher/release/components/releasesConfirm.js

```diff
@@ -3,18 +3,35 @@ import PropTypes from "prop-types";
 import { connect } from "react-redux";
 
 import { cancelPendingReleases } from "../actions/pendingReleases";
+import { releaseRevisions } from "../actions/releases";
 
 class ReleasesConfirm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoading: false
```

Comment:
Possibly would be worth moving it into redux. But that requires all the actions/reducer boilerplate... So I'm fine with treating it out of scope for this PR.